Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -136,7 +171,7 @@ static Boolean __CGPathEqual(CFTypeRef cf1, CFTypeRef cf2) {
 CGMutablePathRef CGPathCreateMutable() {
     __CGPath* mutableRet = __CGPath::CreateInstance();
 
-    mutableRet->initializeGeometries();
+    FAIL_FAST_IF_FAILED(mutableRet->InitializeGeometries());
```

Comment:
Absolutely not.

The user requested a path. We cannot fulfill that. These APIs cannot "Fail" in the classical sense.

---
In reply to: [85839205](https://github.com/Microsoft/WinObjC/pull/1255#discussion_r85839205) [](ancestors = 85839205)